once/__init__.py

```diff
@@ -357,4 +364,6 @@ def __get__(self, obj, cls) -> collections.abc.Callable:
                     bound_func, self.once_factory(), self.fn_type, self.retry_exceptions
                 )
                 self.callables[obj] = callable
+        if self.is_property:
+            return callable()
```

Comment:
Can we just set `func = func.fget()` instead on 345?

Reply:
the class is not bound on 345, so resolving `self` will fail and we cannot call the function yet.